Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml

```diff
@@ -7,7 +7,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 40
-        Structural: 30
+        Piercing: 150 #DeltaV- Damage Bumps ahoy!
```

Comment:
make it AP instead of meme 1 hit crit

Reply:
The reason I didn't make it AP is cause even if it was. At the old damage number it was still shit. It is slow to fire. And doesn't feel like it has the power of a .60 anti material. A rifle this big should down a Greytider running at you with a baseball bat in one

Reply:
Keep its structural